.github/workflows/shellcheck.yml

```diff
@@ -35,9 +35,7 @@ jobs:
             pkg-containers.githubusercontent.com:443
       - uses: actions/checkout@v2
       - name: Install shellcheck
-        run: brew install shellcheck
-        env:
-          HOMEBREW_NO_ANALYTICS: 1
+        run: sudo apt-get install -y shellcheck
```

Comment:
does this install the actual latest version of shellcheck? experience teaches me to be very skeptical of apt

Reply:
I just noticed that shellcheck is an included package in the Ubuntu runner, so this might not be needed at all.